index.bs

```diff
@@ -4854,15 +4856,15 @@ Specifications that Reference Registries</h4>
 
 	<div class=example>
 		For example,
-		“All implementations must implement the <code>Basic-Method</code> as defined in the registry”
-		is not acceptable;
+		 <span class=allow-2119>“All implementations must implement the <code>Basic-Method</code> as defined in the registry”</span>
+		is not acceptable because
 		a change to the definition of the <code>Basic-Method</code> in the registry would then affect conformance.
-		Instead, the requirement must be complete in the specification,
+		Instead, the requirement has to be complete in the specification,
 		directly or by reference to another specification.
 		For example
-		“All implementations must recognize the name <code>Basic-Method</code>,
+		 <span class=allow-2119>“All implementations must recognize the name <code>Basic-Method</code>,</span>
 		and implement it as defined by section yy of IETF RFC xxxx”.
```

Comment:
Very minor nit, but I'd include the whole quote in the `allow-2119` span, since that's the span where the rules are different.
```suggestion
		 <span class=allow-2119>“All implementations must recognize the name <code>Basic-Method</code>,
		and implement it as defined by section yy of IETF RFC xxxx”</span>.
```